### ISSUE TITLE
Link failure in red tile blog (EN)

### ISSUE THREAD

**Issue by MikeMcC399** [corona-warn-app/cwa-website#2976]
## Where to find the issue

- https://www.coronawarn.app/en/blog/2021-12-15-cwa-red-tile-guidance/ "What should you do if you see a red tile?"

## Describe the issue

The link to https://www.zusammengegencorona.de/en/ansteckung-mit-corona-so-wird-das-coronavirus-uebertragen/ in the blog article https://www.coronawarn.app/en/blog/2021-12-15-cwa-red-tile-guidance/ "What should you do if you see a red tile?" fails with a 404 error "Die Seite wurde nicht gefunden".

It seems that the external pages have been rearranged.

The link is only used in the English language version of the blog.

![red tile blog EN](https://user-images.githubusercontent.com/66998419/174796542-0ae47f99-7ff4-4ccf-b86a-8cdea3e443df.jpg)

The German language blog article https://www.coronawarn.app/de/blog/2021-12-15-cwa-red-tile-guidance/ does not contain any link in the equivalent paragraph.

![red tile blog DE](https://user-images.githubusercontent.com/66998419/174796577-3e9f0aed-847c-4a97-881b-79b3ccb2bcf6.jpg)

## Suggested change

Since the German language blog contains no link, I suggest to remove the failing link in the English language blog. That would make the English language blog correspond directly to the German language blog. This is the simplest and cleanest solution.

**Comment by MikeMcC399** [comment]
- I submitted PR #2977 to resolve this issue by removing the link from the English language blog.

**Comment by MikeMcC399** [comment]
@dsarkar 

I checked also on https://www.zusammengegencorona.de/ before I made the submission and I found the same as you that there is a different structure for the English and the German articles.

Blog articles are not normally updated even if the information is out-of-date, unlike FAQ articles, so I didn't want to suggest adding a link to the German blog. Even so, a failing link should not be left in a blog article, so it was good to fix that.

Other information in this article is also out-of-date (such as 14 day risk calculation) and, since the test strategy may also soon change, it could be good to re-issue this article with updates to all topics concerning what to do after a red tile.
